wordpress-mobile/WordPress-iOS
# Plugin manager: Detail view

Introduces the plugin detail view, where you can (de)activate a plugin, toggle autoupdates, visit its homepage, and remove it. It's not a complete match to the initial design, but I think it's good enough for a first PR.

See #7471

## Not implemented yet

- Plugin manual update
- Don't show autoupdates for plugins not in the directory
- Plugin directory link
- Changes in the detail view don't show up in the plugin list until it refreshes
- Persistence/caching

## To test:

Verify that you can:

- Activate/deactivate a plugin
- Toggle autoupdates
- Visit a plugin homepage
- Remove a plugin

Needs review: @elibud
